Tests/SwiftDocCUtilitiesTests/ArgumentParsing/ConvertSubcommandSourceRepositoryTests.swift

```diff
@@ -20,26 +20,35 @@ class ConvertSubcommandSourceRepositoryTests: XCTestCase {
         withExtension: "docc",
         subdirectory: "Test Bundles"
     )!
-    
+
     private let testTemplateURL = Bundle.module.url(
         forResource: "Test Template",
         withExtension: nil,
         subdirectory: "Test Resources"
     )!
-    
+
     func testSourceRepositoryAllArgumentsSpecified() throws {
+        let tempDir = try createTemporaryDirectory(pathComponents: "addFileToCreateValidDirectory")
+        
+        // removing file:// prefix from checkout path because the directory is not
+        // recognized as a valid directory with it
+        let checkoutPath = tempDir.absoluteString
+        let startIndex = checkoutPath.index(checkoutPath.startIndex, offsetBy: 7)
+        let absoluteCheckoutPath = String(checkoutPath[startIndex...])
+
+
         for sourceService in ["github", "gitlab", "bitbucket"] {
             try assertSourceRepositoryArguments(
-                checkoutPath: "checkout path",
+                checkoutPath: absoluteCheckoutPath,
```

Comment:
If you pass the original value (`"checkout path"`) here, then you can use this test to verify that a relative path is transformed into an absolute path. Currently the test would pass even without the changes to the `SourceRepository` initializer.